crates/sui-indexer/src/store/pg_indexer_store.rs

```diff
@@ -95,6 +100,7 @@ const PG_COMMIT_PARALLEL_CHUNK_SIZE: usize = 100;
 // optimistic locking.
 const PG_COMMIT_OBJECTS_PARALLEL_CHUNK_SIZE: usize = 500;
 const PG_DB_COMMIT_SLEEP_DURATION: Duration = Duration::from_secs(3600);
+const GCS_DISPLAY_BUCKET: &str = "mysten-mainnet-display-table";
```

Comment:
We probably want the bucket to be configurable so that different networks don't stomp on each other.